com.unity.render-pipelines.high-definition/package.json

```diff
@@ -3,7 +3,7 @@
   "description": "The High Definition Render Pipeline (HDRP) is a high-fidelity Scriptable Render Pipeline built by Unity to target modern (Compute Shader compatible) platforms. HDRP utilizes Physically-Based Lighting techniques, linear lighting, HDR lighting, and a configurable hybrid Tile/Cluster deferred/Forward lighting architecture and gives you the tools you need to create games, technical demos, animations, and more to a high graphical standard.",
   "version": "9.0.0-preview.11",
   "unity": "2020.1",
-  "unityRelease": "0a23",
+  "unityRelease": "0b1",
```

Comment:
This should be done for all SRP packages, we can do this in a PR after we merge this one